### PR TITLE
Bug 1483055 - Add targeting for pinned sites

### PR DESCRIPTION
**Pull request by k88hudson**
See documentation for description; covered by mochitests, no manual testing needed.

### CODE REVIEW

File: content-src/asrouter/docs/targeting-attributes.md [hunk 1]
```diff
@@ -27,7 +29,7 @@ Please note that some targeting attributes require stricter controls on the tele
 * [topFrecentSites](#topfrecentsites)
 * [totalBookmarksCount](#totalbookmarkscount)
 * [xpinstallEnabled](#xpinstallEnabled)
-* [needsUpdate](#needsupdate)
```
Comment (k88hudson) [inline content-src/asrouter/docs/targeting-attributes.md]: Moved this for alpha-order

File: content-src/asrouter/docs/targeting-attributes.md [hunk 2]
```diff
@@ -210,6 +212,59 @@ localeLanguageCode == "en"
 declare const localeLanguageCode: string;
 ```
 
+### `needsUpdate`
```
Comment (sarracini) [inline content-src/asrouter/docs/targeting-attributes.md]: i think this was copy pasted by accident, `needsUpdate` and `usesFirefoxSync` is here twice now

File: content-src/asrouter/docs/targeting-attributes.md [hunk 3]
```diff
@@ -16,7 +16,9 @@ Please note that some targeting attributes require stricter controls on the tele
 * [firefoxVersion](#firefoxversion)
 * [locale](#locale)
 * [localeLanguageCode](#localelanguagecode)
+* [needsUpdate](#needsupdate)
 * [usesFirefoxSync](#usesfirefoxsync)
```
Comment (sarracini) [inline content-src/asrouter/docs/targeting-attributes.md]: Oh also move this down for alpha too

File: content-src/asrouter/docs/targeting-attributes.md [hunk 4]
```diff
@@ -16,7 +16,9 @@ Please note that some targeting attributes require stricter controls on the tele
...
+* [pinnedSites](#pinnedSites)
```
Comment (sarracini) [inline content-src/asrouter/docs/targeting-attributes.md]: Also small case `sites` here, the linking doesn't work: `#pinnedsites`

**Comment by sarracini** [comment]
Taskcluster is barfing, a couple of them seems related?
```
FAIL Uncaught exception - 2 search topsites are loaded by default
FAIL Uncaught exception - Wait for pinned search topsites
```

**Comment by k88hudson** [comment]
Yep those look real failures

**Pull request merged**